src/Model/MaterialModel_Linear_Elasticity.cpp

```diff
@@ -0,0 +1,65 @@
+#include "MaterialModel_Linear_Elasticity.hpp"
+
+MaterialModel_Linear_Elasticity::MaterialModel_Linear_Elasticity(
+    const double &in_modulus_E, const double &in_nu )
+: modulus_E( in_modulus_E ), nu( in_nu )
+{
+}
```

Comment:
simpler way.

Reply:
This constructor has been moved to the header file.